pkg/kms/webkms/remotekms.go

```diff
@@ -116,17 +100,24 @@ func New(provider storage.Provider, client *http.Client, marshaller marshalFunc)
 func CreateKeyStore(store storage.Store, httpClient *http.Client, keyserverURL, controller string,
```

Comment:
`CreateKeyStore` is not a part of `kms.KeyManager` API. Should it be here?

Reply:
it's part of webkms api though.. it's kept here to maintain webkms interoperability even though it's not part of kms.KeyManager API